cli/tests/default/e2e-other/test_taint_intrafile.py

```diff
@@ -1,61 +0,0 @@
-import pytest
```

Comment:
That's the only test that actually depend on semgrep-core-proprietary, but this test should be moved to the Pro repo!

Reply:
Can you move this test to the pro repo, so that we don't drop the ball on migrating it?